### PR TITLE
Bump OCP version to 4.7

### PR DESCRIPTION
**Pull request by periklis**
### Description
This PR ensures builds for elasticsearch-operator based on the downstream ART pipeline. It supersedes #517 .
 
/cc @blockloop 
/assign  @ewolinetz 

### Links
- Bugzilla:  https://bugzilla.redhat.com/show_bug.cgi?id=1887361
- Relevant PRs: #517, openshift/cluster-logging-operator#736 (Hard dependency for `ci/prow/cluster-logging-operator-e2e`)

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/approve
/lgtm

**Comment by ewolinetz** [comment]
/hold
please update skip range to start at 4.5.0

**Comment by periklis** [comment]
> /hold
> please update skip range to start at 4.5.0

Here we go!

File: manifests/elasticsearch-operator.package.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #! package-manifest: ./deploy/chart/catalog_resources/rh-operators/elasticsearch-operator.v4.6.0.clusterserviceversion.yaml
```
Comment (vimalk78) [inline manifests/elasticsearch-operator.package.yaml:1]: need to change this comment line?

**Comment by ewolinetz** [comment]
/hold
to investigate why this is failing

**Comment by periklis** [comment]
> /hold
> to investigate why this is failing

This is failing because the cluster-logging-operator mounts the source of elasticsearch operator to run its E2E testing. Since CLO expects a 4.7 EO, we have a circular dep here. As with the 4.6 bump we need to manual merge this. (cc @jcantrill )

**Pull request merged**